### ISSUE TITLE
Light theme for dashboard with minimal coloring

### ISSUE THREAD

**Issue by Darin755** [netbirdio/netbird#2918]
**Is your feature request related to a problem? Please describe.**
Right now the dashboard only has one dark theme that is over the top and sometimes harder to read. (At least in my opinion) The dark theme is really unnecessary and I would prefer to have something much more basic and functional. The old dashboard was nice because it was simple and to the point. The new page is fine but it would be nice if Netbird defaulted to something more traditional. 

**Describe the solution you'd like**
Ideally it should light with strong contrast and little additional effects like transparency. Examples:
![1883 blog-2689499218](https://github.com/user-attachments/assets/25bad306-7266-4e32-b936-4ed639867367)
![kubernetes-dashboard-ui-2899646722](https://github.com/user-attachments/assets/ea3f87bd-2f74-471c-972c-24ae1ed85fea)
![server-manager-screen-2799000919](https://github.com/user-attachments/assets/0ee90f0d-4f0b-4496-81c9-bc2714f23921)

**Describe alternatives you've considered**
Right now it isn't a deal breaker not to have a light theme. I can definitely tell there was some serious effort put in to making the current dashboard. I think it is already more user friendly than Tailscale. 

**Additional context**
Maybe remove the theme switcher button until you have a working light theme. I like the idea but right now it doesn't serve much purpose.

**Comment by mellow65** [comment]
Yes please a light mode would be amazing!! My eyes have a real hard time with dark modes and essentially all the letters get washed out by the black background.  I struggle to read the text.

**Comment by markcst** [comment]
Please, introduce this light mode, hopefully with a classic (follow) system mode, in order to have
- System theme (follow system's)
- Light theme
- Dark theme

**Comment by lixmal** [comment]
Duplicate of https://github.com/netbirdio/dashboard/issues/412

**Comment by TimoKunze** [comment]
I would love to see a light theme. Or even better: Simply follow the system settings. I dislike dark mode and find it hard to read.

**Comment by TechHutTV** [comment]
<img width="480" alt="Image" src="https://github.com/user-attachments/assets/f89d3e29-ba15-4b34-97bf-f3ef558f8930" />

Light mode is now included in the [latest RC](https://github.com/netbirdio/netbird/releases/tag/v0.79.0-rc.1) you can also view https://github.com/netbirdio/netbird/commit/ec0c36b0e7b34fd17948ff3fcd7c77d50104750e. Closing as completed. Light mode in dashboard will follow! 

cc. @markcst @mellow65 @TimoKunze @Darin755
